features/specs/gezag-minderjarige/verblijf-buitenland.feature

```diff
@@ -0,0 +1,280 @@
+      Dan is het gezag over 'Bert' gezamenlijk ouderlijk gezag met ouder 'Gerda' en ouder 'Aart'
+
+    Voorbeeld: minderjarige is geëmigreerd naar het buitenland en heeft twee ouders die met elkaar gehuwd waren
+      Gegeven 'Gerda' en 'Aart' zijn met elkaar gehuwd
+      En 'Gerda' en 'Bert' zijn 12 jaar geleden gescheiden
+      En persoon 'Bert'
+      * heeft 'Aart' en 'Gerda' als ouders
+      * is 15 jaar geleden geboren
+      * is geboren in Nederland
+      En 'Bert' is 10 jaar geleden geëmigreerd naar Duitsland
+      Als gezag wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' gezamenlijk ouderlijk gezag met ouder 'Gerda' en ouder 'Aart'
+
+    Voorbeeld: ouders van minderjarige waren nooit met elkaar gehuwd en minderjarige is geboren na 1-1-2023 en daarna geëmigreerd
+      Gegeven persoon 'Bert'
+      * heeft 'Aart' en 'Gerda' als ouders
+      * is geboren op 6-1-2023
+      En 'Bert' is 1 jaar geleden geëmmigreerd naar Duitsland
+      Als gezag wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' gezamenlijk ouderlijk gezag met ouder 'Gerda' en ouder 'Aart'
+
+    Voorbeeld: ouders van minderjarige waren nooit met elkaar gehuwd en minderjarige is erkend na 1-1-2023 en daarna geëmigreerd
+      Gegeven persoon 'Bert'
+      * heeft 'Gerda' als ouder
+      * is geboren op 14-10-2019
+      En 'Bert' is erkend door 'Aart' op 6-1-2023
+      En 'Bert' is 1 jaar geleden geëmmigreerd naar Duitsland
+      Als gezag wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' gezamenlijk ouderlijk gezag met ouder 'Gerda' en ouder 'Aart'
+
+  Regel: Het gezag kan worden bepaald voor een minderjarige die ooit Nederland als vaste verblijfplaats heeft gehad en waarvoor van rechtswege gezamenlijk gezag is bepaald
+
+    Voorbeeld: minderjarige is in het buitenland geboren en heeft één ouder die ten tijde van de geboorte van minderjarige gehuwd was
+      Gegeven 'Gerda' en 'Zoe' zijn 20 jaar geleden gehuwd
+      En persoon 'Bert'
+      * heeft 'Gerda' als ouder
+      * is geboren in Duitsland
+      * is 3 jaar geleden geïmmigreerd
+      En 'Gerda' en 'Zoe' zijn 2 jaar geleden gescheiden
+      Als gezag wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' gezamenlijk gezag met ouder 'Gerda' en derde 'Zoe'
+
+    Voorbeeld: minderjarige heeft één ouder die ten tijde van de geboorte van minderjarige gehuwd was en minderjarige is geëmigreerd
+      Gegeven 'Gerda' en 'Zoe' zijn 20 jaar geleden gehuwd
+      En persoon 'Bert'
+      * heeft 'Gerda' als ouder
+      * is geboren in Nederland
+      En 'Bert' is 5 jaar geleden geëmmigreerd naar Duitsland
+      Als gezag wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' gezamenlijk gezag met ouder 'Gerda' en derde 'Zoe'
+
+  Regel: Het gezag kan niet worden bepaald voor een minderjarige die een vaste verblijfplaats in het buitenland heeft of heeft gehad en waarvoor van rechtswege eenhoofdig ouderlijk gezag is bepaald
```

Comment:
Suggestie: Het gezag kan niet worden bepaald voor een minderjarige die ooit een vaste verblijfplaats in het buitenland heeft gehad, en eenhoofdig ouderlijk gezag van rechtswege.